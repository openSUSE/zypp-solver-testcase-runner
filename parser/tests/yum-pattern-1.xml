<?xml version="1.0" encoding="UTF-8"?>
<patterns  xmlns="http://linux.duke.edu/metadata/patterns">
  <pattern>
    <patternid>foo</patternid>
    <name>foobar</name>
    <name lang='en.US'>foobar</name>
    <default>false</default>
    <uservisible>true</uservisible>
    <description>This is my pattern, it is soooooooo coool!</description>
    <description lang='en.US'>Duh</description>
    <patternlist>
       <metapkg type="mandatory">otherpattern</metapkg>
       <metapkg type="optional">stillother</metapkg>
       <metapkg type="default">stillmore</metapkg>
       <metapkg>other</metapkg>
    </patternlist>
    <packagelist>
       <packagereq type="mandatory" epoch="0" ver="1" rel="1">pkgname</packagereq>
       <packagereq epoch="1" ver="2" rel="0">otherpkgname</packagereq>
    </packagelist>
  </pattern>
</patterns>

