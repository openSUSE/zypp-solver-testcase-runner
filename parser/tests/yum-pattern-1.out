-------------------------------------------------
Pattern Data: 
pattern-id: 'foo'
name:
foobar
[en.US] foobar
default: 'false'
user-visible: 'true'
description:
This is my pattern, it is soooooooo coool!
[en.US] Duh
patternlist:
type: mandatory, name: otherpattern
type: optional, name: stillother
type: default, name: stillmore
type: , name: other
packageList:
[mandatory] pkgname 0-1-1
[] otherpkgname 1-2-0
Exit Code: 0
